documentation/markdown/architecture/features/protocol/parsing.md

```diff
@@ -15,13 +16,14 @@ flowchart TD
 A `ParsingHttpHandler` handles both the parsing of the input data, and the serializing of the output data.
 It follows these 3 steps:
 
-1. Use the `RequestParser` to convert the incoming data into an `Operation`.
-2. Send the `Operation` to the `AuthorizingHttpHandler` to receive either a `Representation` if the operation was a success,
-   or an `Error` in case something went wrong.
-   * In case of an error the `ErrorHandler` will convert the `Error` into a `ResponseDescription`.
-3. Use the `ResponseWriter` to output the `ResponseDescription` as an HTTP response.
+1.  Use the `RequestParser` to convert the incoming data into an `Operation`.
+2.  Send the `Operation` to the `AuthorizingHttpHandler` to receive either a `Representation` if the operation was a success,
+    or an `Error` in case something went wrong.
+    * In case of an error the `ErrorHandler` will convert the `Error` into a `ResponseDescription`.
+3.  Use the `ResponseWriter` to output the `ResponseDescription` as an HTTP response.
```

Comment:
Why the extra spaces? Going to not add this comment anymore but I noticed several similar situations below.

Reply:
Short answer: when item of multi is more then one paragraph, there should be 2 spaces for ol and 3 spaces for ul
Full style guide entry: https://cirosantilli.com/markdown-style-guide/#spaces-after-list-marker

mkdocs adheres to this styling themselves in their repository.

Reply:
The 4 spaces I could get, but this is just weird 😄. Can this linter automatically fix issues like this?

Looking at the documentation `list-space:1` would not have this issue. Can we also use this instead or does mkdocs require `list-space:mixed`?

Reply:
Its not that weird from a readability point of view :) It looks way nicer and more readable in the editor that way. I don't actually think `mkdocs` has to have this to render its content though. Might be fine to just use list-space one, but enforcing a 4 spaces indent between list bullet/number and the list item multiline list just seems nicer to me personally.

As to fixing these errors, yes the tool can help in most cases such as this one:
`npx markdownlint-cli2-fix <markdown-file>` tries to fix all issues. Or you can use vim or vscode extension to format on save.

Maybe best to add a `fix:markdown` script or something similar?

Reply:
A `lint:markdown:fix` (or `lint:markdown-fix`, not sure what the expectations are about the `:`) would be nice indeed. Have the name start with the same string so they appear next to each other in the package.json.